interchaintest/conformance_test.go

```diff
@@ -0,0 +1,69 @@
+package interchaintest_test
+
+import (
+	"context"
+	"testing"
+
+	interchaintest "github.com/strangelove-ventures/interchaintest/v7"
+	"github.com/strangelove-ventures/interchaintest/v7/conformance"
+	"github.com/strangelove-ventures/interchaintest/v7/ibc"
+	"github.com/strangelove-ventures/interchaintest/v7/testreporter"
+	"go.uber.org/zap/zaptest"
+)
+
+func TestConformance(t *testing.T) {
+	// Arrange
+	numOfValidators := 2
+	numOfFullNodes := 0
+
+	cf := interchaintest.NewBuiltinChainFactory(zaptest.NewLogger(t), []*interchaintest.ChainSpec{
+		latestFinschiaChain(numOfValidators, numOfFullNodes),
+		builtinCosmosChainV13(numOfValidators, numOfFullNodes),
+	})
+
+	rlyFactory := interchaintest.NewBuiltinRelayerFactory(
+		ibc.CosmosRly,
+		zaptest.NewLogger(t),
+	)
+
+	// Act & Assert
+	conformance.Test(t, context.Background(), []interchaintest.ChainFactory{cf}, []interchaintest.RelayerFactory{rlyFactory}, testreporter.NewNopReporter())
+}
+
+func latestFinschiaChain(numOfValidators, numOfFullNodes int) *interchaintest.ChainSpec {
+	return &interchaintest.ChainSpec{
+		Name:      "finschia",
+		ChainName: "finschia",
```

Comment:
How about changing other name like `finschia-2`?

Reply:
Is `finschia-2` for ChainName?
What is the proper Name and ChainName for Finschia?
cosmos-hub is as follows.

```
&interchaintest.ChainSpec{
		Name:          "gaia",
		ChainName:     "cosmoshub-2",
		Version:       "v13.0.1",
		NumValidators: &numOfValidators,
		NumFullNodes:  &numOfFullNodes,
	}
```